Core/PaymentMethodHelper.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Core;
+
+use OxidEsales\Eshop\Core\Model\ListModel;
+
+use Wirecard\Oxid\Extend\Model\Payment;
+
+/**
+ * Helper class to handle payment methods
+ *
+ * @since 1.0.0
+ */
+class PaymentMethodHelper
+{
+    /**
+     * Returns a payment with the selected id.
+     *
+     * @param string $sPaymentId
+     *
+     * @return Payment
+     *
+     * @since 1.0.0
+     */
+    public static function getPaymentById($sPaymentId)
```

Comment:
Great that you did this now, please also refactor the old code and use it everywhere you can. (NotifyHandler.php, PaymentGateway.php, Order.php, OrderController.php, PaymentMain.php)

Reply:
It is now used in all the classes (thanks for the list btw 👍 )